runbooks/workflow/upgrade-metabase.md

```diff
@@ -0,0 +1,48 @@
+# Upgrading Metabase
+
+Metabase upgrades happen in two phases: validation and release.
+
+## Validation
+
+### Prerequisites
+
+- Lens (https://k8slens.dev)
+- Restic (https://restic.net)
+- A copy of the secrets for the GCloud service account
+
+### Service Credentials
+
+1. Log into GCloud: `gcloud auth login`
+2. List the clusters you can access: `gcloud container clusters list`
+3. Pull Kubernetes credentials for the `data-infra-apps` cluster to your local machine: `gcloud container clusters get-credentials data-infra-apps --location us-west1`
+
+### Replicating the database
+
+1. Retrieve environment variables for one of the `metabase-test` `postgresql-backup` pods
+2. Set the `GOOGLE_APPLICATION_CREDENTIALS`, `GOOGLE_PROJECT_ID`, `RESTIC_REPOSITORY` and `RESTIC_PASSWORD` environment variables in a shell
+3. List available Restic snapshots using `restic snapshots` and note the latest SHA and filename
+4. Download the latest snapshot `restic dump -t <OUTPUT> <SHA> <FILENAME>`
```

Comment:
let's just fill in `<FILENAME>` in the example and maybe a `/tmp` path for output since it can be confusing what to put there and the values can be static